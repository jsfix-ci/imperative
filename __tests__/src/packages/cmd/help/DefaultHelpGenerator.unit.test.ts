import { DefaultHelpGenerator } from "../../../../../packages/cmd/src/help/DefaultHelpGenerator";


describe("Test DefaultHelpGenerator", () => {
    test("escapeMarkdown: removing ansi escape codes", async () => {
        // @ts-ignore
        const result = DefaultHelpGenerator.prototype.escapeMarkdown(
            `Specifies whether to verify that the objects to be created do not exist\
 on the Db2 subsystem and that the related objects that are required for successful creation\
 of the objects exist on the Db2 subsystem or in the input DDL.
 \n \u001b[90m Default value: no \u001b[0m`
        );

        expect(result).toMatchSnapshot();
    });
});
