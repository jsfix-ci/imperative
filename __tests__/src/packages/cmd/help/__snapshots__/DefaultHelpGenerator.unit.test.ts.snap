// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test DefaultHelpGenerator escapeMarkdown: removing ansi escape codes 1`] = `
"Specifies whether to verify that the objects to be created do not exist on the Db2 subsystem and that the related objects that are required for successful creation of the objects exist on the Db2 subsystem or in the input DDL\\\\.
 
  Default value: no "
`;
